packages/@sanity/migrate/src/mutations/operations/creators.ts

```diff
@@ -62,6 +84,12 @@ export const inc = <const N extends number = 1>(amount: N = 1 as N): IncOp<N> =>
  * @returns A `dec` operation.
```

Comment:
```suggestion
 * @returns An decrementation operation for numeric values
```

Reply:
Isn't it more commonly referred to as "a decrement" operation? (Realized too late it should probably also be "An increment operation" above, if so)